### PR TITLE
[zh]update verify-signed-images.md

### PR DESCRIPTION
**Pull request by tengqm**
Signed-off-by: bin.hui <bin.hui@daocloud.io>
Co-authored-by: Qiming Teng <tengqm@outlook.com>
closes: #34221

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-images.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+`COSIGN_EXPERIMENTAL=1` 用于对以 `KEYLESS` 模式签名的镜像进行验证。想要进一步了解 `KEYLESS`，请参考
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)。
+{{< /note >}}
+
+<!--
+### Verifying images for all control plane components
+
+To verify all signed control plane images, please run this command:
+-->
+### 验证所有控制平面组件镜像
```
Comment (ydFu) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-images.md:70]:
```suggestion
### 验证所有控制平面组件镜像  {#verifying-images-for-all-control-plane-components}
```

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-images.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+e.g. [conformance image](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/image/README.md))
+, signatures can also be verified at deploy time using
+[cosigned](https://docs.sigstore.dev/cosign/kubernetes/#cosigned-admission-controller)
+admission controller. To get started with `cosigned` here are a few helpful
+resources:
+
+* [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/cosigned)
+* [Configuration Options](https://github.com/sigstore/cosign/tree/main/config)
+-->
+## 使用准入控制器验证镜像签名
```
Comment (ydFu) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-images.md:110]:
```suggestion
## 使用准入控制器(Admission Controller)验证镜像签名   {#verifying-image-signatures-with-admission-controller}
```

**Comment by ydFu** [comment]
LGTM
/lgtm

**Comment by mengjiao-liu** [comment]
/approve
Thanks!

**Pull request merged**